lib/cmds/space_cmds/create.ts

```diff
@@ -103,11 +106,7 @@ the Pricing page: https://www.contentful.com/pricing/?faq_category=payments&faq=
   }
 
   const intentSystem = new IntentSystem()
-  intentSystem.addHandler(
-    createSpaceIntents({
-      skipConfirm: yes
-    })
-  )
+  intentSystem.addHandler(createSpaceIntents())
```

Comment:
Why did we remove `skipConfirm: yes` here?

Reply:
my bad, forgot to comment about this one, it is because the `createSpaceIntents` doesn't accept any parameters, I saw that the previous version of the method accepted, but it was removed from there, but not here:
https://github.com/contentful/contentful-cli/blob/master/lib/core/event-handlers/intents/create-space-handler.js